spec/rails_helper.rb

```diff
@@ -31,6 +31,7 @@ module Features
 end
 
 ActiveRecord::Migration.maintain_test_schema!
+ActiveSupport::Deprecation.silenced = true
```

Comment:
Can we do this around the specific test instead of globally? We don't want to suppress all deprecation warnings, some of which may come from `ActiveSupport`/Rails itself.

This would also help show which test is there to ensure we're doing this in a backwards compatible way, as looking at the changes here it's hard to tell what we'll need to remove when we change the feature completely.

Reply:
Moved deprecation silencing to the spec